profiles/common/modules/custom/ecas/includes/ecas.inc

```diff
@@ -402,22 +402,20 @@ function ecas_ecas_sync_user_info($user, array $user_info, array &$edit, array $
   if (!empty($user_info['cas:groups']) && !empty($user_info['cas:groups']['cas:group'])) {
     $user_cas_groups = (array) $user_info['cas:groups']['cas:group'];
   }
-
-  $cem_role = user_role_load_by_name(CCE_BASIC_CONFIG_CEM_ROLE_NAME);
+  $administrator_role = user_role_load_by_name("administrator");
   $user_roles = (isset($user->roles)) ? $user->roles : array();
   // By default, the role is not set for any user automatically.
   // Then, unset it by default and reassign it later if the user has the
   // EU login group.
-  unset($user_roles[$cem_role->rid]);
+  unset($user_roles[$administrator_role->rid]);
```

Comment:
Admin role is removed but you should add/keep this role if the user is already grant as Administrator

Reply:
This has been removed